test/index.html

```diff
@@ -18,7 +18,10 @@
 <body>
   <script>
     WCT.loadSuites([
-      'lazy-imports_test.html',
+      '1.x/lazy-imports_test.html?dom=shadow',
```

Comment:
Oh nvm my previous message, I didn't have enough context.
So to know if we're using Polymer 2.0 we can check for `Polymer.Element`.
I'd suggest to keep the hybrid tests in the `test/` folder, and create a `test/2.0/`:
- lazy-imports_test.html?wc-shadydom=true&wc-ce=true
- lazy-imports_test.html?dom=shadow
- 2.0/lazy-imports_test.html?wc-shadydom=true&wc-ce=true
- 2.0/lazy-imports_test.html